### ISSUE TITLE
Getting continuous outputs

### ISSUE THREAD

**Issue by danhardman** [NOX73/go-neural#8]
Okay, actual question with the library now, not just me being stupid with neural networks.

How can I use this to predict continuous outputs? I want to use this to predict the time it takes to do something, based on 3 variables so classes don't really work.

**Comment by NOX73** [comment]
Three inputs and one output? then closer to 1 then more likely you should do your action

**Comment by danhardman** [comment]
Wouldn't really work in this situation, it the point of continuous output is that you can get real values from it, as opposed to thresholding the value to make it Off/On.

Here's a sample of the [temperature_data.txt](https://github.com/NOX73/go-neural/files/122944/temperature_data.txt) I'm using where `inside_temperature`, `outside_temperature` and `target_temperature` are the 3 inputs and the desired output of each record is the `minutes_taken` column. Is there a way I can use this library to output values like the `minutes_taken` column? 

I think I'd need to play around with how the activation functions work to increase the range on the sigmoid functions to the scale of my desired output values, as opposed to [0-1] so I can have [0-MAX_OUTPUT] where MAX_OUTPUT would be calculated from finding the largest value in the training dataset possibly?

**Comment by NOX73** [comment]
I think you can just divide all `minutes_taken` column values by `MAX_OUTPUT` and try to learn your net with this values.

**Comment by danhardman** [comment]
I hadn't thought about going at it that way. I'll mock something up later on tonight! Thanks for replying quickly!

**Comment by danhardman** [comment]
Just thought I'd give you an update on this. Normalising the ideal output before learning and de-normalising after calculating sorted this. Predictions are pretty awesome now so all is good!
